python/tflite_micro/README.md

```diff
@@ -1,25 +1,127 @@
-# TFLM Python Interpreter
+# The `tflite_micro` Python Package
 
-The TFLM interpreter can be invoked from Python by using the Python interpreter
-wrapper in this directory.
+This directory contains the `tflite_micro` Python package. The following is
+mainly documentation for its developers.
 
-## Usage
+The `tflite_micro` package contains a complete TFLM interpreter built as a
+CPython extension module. The build of simple Python packages may be driven by
+standard Python package builders such as `build`, `setuptools`, and `flit`;
+however, as TFLM is first and foremost a large C/C++ project, `tflite_micro`'s
+build is instead driven by its C/C++ build system Bazel.
 
-There are two ways to import the Python wrapper, either by using Bazel/Blaze, or
-in near future by installing a PyPi package.
+## Building and installing locally
 
-### Bazel
+### Building
 
-#### Build
+The Bazel target `//python/tflite_micro:whl.dist` builds a `tflite_micro`
+Python *.whl* under the output directory `bazel-bin/python/tflite_micro/whl_dist`. For example:
+```
+% bazel build //python/tflite_micro:whl.dist                                                                   ~/wetu/tflite-micro
```

Comment:
Do we need that trailing path?

Reply:
Doh, that's a copy-and-paste error I didn't catch. Will remove.